openwisp-config/files/openwisp.agent

```diff
@@ -460,8 +463,8 @@ test_configuration() {
 }
 
 perform_default_test() {
-	# max 3 attempts to get checksum
-	for i in $(seq 1 3); do
+	# by default max 3 attempts to get checksum
```

Comment:
this comment seems somewhat outdated, I would change it to:

```
# the default test is performed several times, as indicated by $TEST_RETRIES
```

Reply:
Changed.